beacon-chain/cache/aibench_test.go

```diff
@@ -0,0 +1,107 @@
+package cache
```

Comment:
please choose better filename. benchmarks_test.go is OK, but the `ai` prefix is not helpful description.